packages/@guardian/source-react-components-development-kitchen/components/divider/Divider.tsx

```diff
@@ -1,18 +1,19 @@
 import { css, SerializedStyles } from '@emotion/react';
-
 import { border, text } from '@guardian/src-foundations/palette';
 import { space } from '@guardian/src-foundations';
 import { textSans } from '@guardian/src-foundations/typography';
-
 import { partialStyles, fullStyles } from './styles';
+
+type Sizes = 'full' | 'partial' | 'fit';
+type Spaces = 'tight' | 'loose' | 'none';
```

Comment:
is the convention to not use plural here? I feel like just `Size` and `Space` is more consistent with other places?

Reply:
If we need to pass in `spaceAbove: 'none';` does that not mean that the values we're using - _somewhere_ - are wrong?

Ideally, the goal should be consistent spacing above the divider. If there's a unique case where, for a specific reason, we need a different value we could use `cssOverrides` to break out of the common values?

Reply:
Agreed re: plural naming

Having the ability to turn off all margin around the divider to leave spacing up to the end user strikes me as a good potential solution, as there is a negative margin-bottom too which we'd need to add a `spaceBelow` prop for, which seems messy

Reply:
I think we discussed this when first creating this component. It's a tricky one. On one hand, the division of space is intrinsic to what a 'divider' is so the size of the margin is part of its style. But in other cases we don't set these vertical margins, expecting the platforms to manage this themselves.

My feeling here is that platform specific margins are different to this particular margin. On Dotcom there is a uniform 12px space above each element on an article which is managed by a wrapper component `Figure`. But a divider needs more space and it would feel wrong to try and build that logic into `Figure` to do this.

I wonder if the real question is: Why do you need a different amount of space here? Why isn't 'tight' or 'loose' right for your use case? Could we adjust the designs for where this component will be used to align with the common standard?